packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -316,7 +316,7 @@ const conf = (module.exports = convict({
     },
     clientId: {
       default:
-        '210899493109-gll5587a3bo8huare772alo08734o4kh.apps.googleusercontent.com',
+        '210899493109-u0erdakhegg6b572faenm3mn9hf4mdp8.apps.googleusercontent.com',
```

Comment:
I think this is repeated 3&times; Not sure if there is a better way to share it (it's awkward since it's just a default fallback value for an ENV var, so extremely low priority); although easy to see how this could potentially get out of sync:

```sh
git grep -En "'.*\.apps\.googleusercontent\.com'" packages | cat

packages/fxa-admin-server/src/config/index.ts:471:        '210899493109-u0erdakhegg6b572faenm3mn9hf4mdp8.apps.googleusercontent.com',
packages/fxa-auth-server/config/index.ts:201:        '210899493109-u0erdakhegg6b572faenm3mn9hf4mdp8.apps.googleusercontent.com',
packages/fxa-content-server/server/lib/configuration.js:319:        '210899493109-u0erdakhegg6b572faenm3mn9hf4mdp8.apps.googleusercontent.com',
```

```sh
git grep -Eoh "'.*\.apps\.googleusercontent\.com'" packages | sort | uniq

'210899493109-u0erdakhegg6b572faenm3mn9hf4mdp8.apps.googleusercontent.com'
```